plugin/src/main/resources/jenkins/plugins/openstack/compute/slaveopts/LauncherFactory/KAFKA/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="KAFKA remote agent launch">
+        <f:entry>
+            <p>Jenkins will wait for the Kafka connection to be initiated from the slave itself. Note this plugin does not initiate
+            that. See help for <i>User Data</i> field for more information on the available options.</p>
```

Comment:
We might need to massage this a bit, by ideally moving the details here. For now, JNLP was the only inbound connection supported so the user-data config mentioned that only. Kafka seems to require specific launching instructions, that needs to be documented here (or at least link to plugin documentation). Also, there will be no way for the other plugin to contribute documentation for User Data, so this might be the only place to document it.